patches/0001-archive-tar-do-not-populate-user-group-names.patch

```diff
@@ -11,10 +11,6 @@ on NULL pointer dereference).
 
 For much more details, see https://github.com/golang/go/issues/23265
```

Comment:
This link is also misleading, but at least it's not quite as confusing. :+1: :heart: